includes/definitions/discovery/vrp.yaml

```diff
@@ -33,7 +33,9 @@ modules:
                 -
                     oid:
                         - entPhysicalName
+                        - hwBoardName
```

Comment:
You should replace "hwBoardName" by "hwEntityStateTable" (which is the parent oid) to avoid double snmpwalk. "hwEntityStateTable" is anyway walked later in the file.